Source/EventFlow.EntityFramework/Extensions/ModelBuilderExtensions.cs

```diff
@@ -34,6 +34,7 @@ public static ModelBuilder AddEventFlowEvents(this ModelBuilder modelBuilder)
             var eventEntity = modelBuilder.Entity<EventEntity>();
             eventEntity.HasKey(e => e.GlobalSequenceNumber);
             eventEntity.HasIndex(e => new {e.AggregateId, e.AggregateSequenceNumber}).IsUnique();
+            eventEntity.HasIndex(e => e.Confirmed);
```

Comment:
we also need second index {e.AggregateId, e.Confirmed} for LoadUnconfirmedEventsAsync(id) method